cmd/spicedb/serve.go

```diff
@@ -82,6 +82,7 @@ func registerServeCmd(rootCmd *cobra.Command) {
 	serveCmd.Flags().Duration("datastore-gc-interval", 3*time.Minute, "amount of time between passes of garbage collection (postgres driver only)")
 	serveCmd.Flags().Duration("datastore-gc-max-operation-time", 1*time.Minute, "maximum amount of time a garbage collection pass can operate before timing out (postgres driver only)")
 	serveCmd.Flags().Duration("datastore-revision-fuzzing-duration", 5*time.Second, "amount of time to advertize stale revisions")
+	serveCmd.Flags().Duration("datastore-follower-read-delay-duration", 0*time.Second, "amount of time to use as a delay to enable follower reads (cockroach driver only)")
```

Comment:
"amount of time to subtract from non-sync revision timestamps to ensure they are sufficiently in the past to enable follower (historical) reads (cockroach driver only)"

Reply:
We should also ideally link to documentation so people know how to configure this

Reply:
We should probably also default it to 4.8s

Reply:
It is only applied to the crdb datastore init so that's reasonable.